assets/activation-scripts/activate

```diff
@@ -167,9 +166,10 @@ fi
 _FLOX_ENV="$($_dirname -- "${BASH_SOURCE[0]}")"
 export FLOX_ENV="${FLOX_ENV:-$_FLOX_ENV}"
 
-# Now that FLOX_ENV has been updated by way of getopt, fix the env accordingly.
 # shellcheck disable=SC1090
-source <("$_flox_env_helper" "bash")
+source <("$_flox_activations" set-env-dirs --shell bash --flox-env "$FLOX_ENV" --env-dirs "${FLOX_ENV_DIRS:-}")
+# shellcheck disable=SC1090
+source <("$_flox_activations" fix-paths --shell bash --env-dirs "$FLOX_ENV_DIRS" --path "$PATH" --manpath "${MANPATH:-}")
```

Comment:
question nonblocking: are the cases where FLOX_ENV_DIRS and MANPATH default to `""` handled correctly? Are those worth testing?

Reply:
Yes, but I added some tests to catch regressions anyway.